app-modules/form/src/Http/Controllers/FormWidgetController.php

```diff
@@ -254,4 +271,67 @@ public function store(
             'message' => 'Form submitted successfully.',
         ]);
     }
+
+    public function registerProspect(RegisterProspectRequest $request, Form $form): JsonResponse
+    {
+        $data = $request->validated();
+
+        $prospect = Prospect::query()
+            ->make([
+                'email' => $data['email'],
+                'first_name' => $data['first_name'],
+                'last_name' => $data['last_name'],
+                'full_name' => "{$data['first_name']} {$data['last_name']}",
+                'preferred' => $data['preferred'] ?? null,
+                'bitrhdate' => $data['bitrhdate'],
+                'mobile' => $data['mobile'],
+                'address' => $data['address'],
+                'address_2' => $data['address_2'],
+                'city' => $data['city'],
+                'state' => $data['state'],
+                'postal' => $data['postal'],
```

Comment:
Can this be simplified?

```suggestion
                ...$data,
                'full_name' => "{$data['first_name']} {$data['last_name']}",
```